cmd/nerdctl/container_inspect.go

```diff
@@ -69,18 +69,13 @@ func containerInspectAction(cmd *cobra.Command, args []string) error {
 		OnFound: f.Handler,
 	}
 
-	var errs []error
-	for _, req := range args {
-		n, err := walker.Walk(ctx, req)
-		if err != nil {
-			errs = append(errs, err)
-		} else if n == 0 {
-			errs = append(errs, fmt.Errorf("no such object: %s", req))
-		}
-	}
-	if len(errs) > 0 {
-		return fmt.Errorf("%d errors: %v", len(errs), errs)
+	n, err := walker.Walk(ctx, args[0])
```

Comment:
Why only args[0]

Reply:
I forgot that we can inspect an image or a container directly 😅